### PR TITLE
33 fulfill columns of country informations

### PR DESCRIPTION
**Pull request by FVarlet**
Proposed changes don't include the case 'Koweit-Saudi Arabia' yet

### CODE REVIEW

File: main_script.py [hunk 1]
```diff
@@ -88,7 +88,21 @@ def concat_dataframe_into_excel(fpath: str):
               "Skipping Step3 and Step4\n")
 
     # Step5 : Scrape country informations
-    df_countries = create_country_table()
+    url_undata = "https://data.un.org/"
```
Comment (mnicolleUTC) [inline main_script.py:91]: Is it possible for you to include the scrapping from data.un in the function create_country_table(). Like this it simplify the main script for newcomers and it include all step of country creation into the function create_country_table(). 
Would be preferable but if not possible (for time limitation) not a problem :)

File: data_cleaned/country_informations.csv [hunk 2]
```diff
@@ -1,49 +1,3523 @@
...
-Türkiye
-Ukraine
-United Arab Emirates
-United Kingdom
-United States
-Uzbekistan
-Venezuela
-Zimbabwe
+Country_source_CB,Year,Category,Series,Value,Footnotes,Source
+Algeria,2010,"Population, density and surface area",Population mid-year estimates (millions),35.86,,"United Nations Population Division, New York, World Population Prospects: The 2022 Revision, last accessed July 2022."
```
Comment (mnicolleUTC) [inline data_cleaned/country_informations.csv:2]: I'm not sure but it seems, there is a formating error of the informations in the output csv. We can talk about it on Slack if you want

**Review by mnicolleUTC — COMMENTED** [review]
1 major and 1 minor issues otherwise it's perfect :)

**Comment by Nathanlauga** [comment]
Update postprocessing to get one row per country and keep only wanted KPI (identied with dataviz team)

File: data_cleaned/country_informations.csv (hunk 2 above)
Comment (Nathanlauga) [reply data_cleaned/country_informations.csv:2]: Postporcess OK

File: main_script.py (hunk 1 above)
Comment (Nathanlauga) [reply main_script.py]: Remove comments because it's already put into the countries.py file

**Review by Nathanlauga — APPROVED** [review]
Great job! TODO in the future, see how to automate the scrapping of the UN data.

**Pull request merged**